content/en/docs/writing-policies/policy-settings.md

```diff
@@ -13,7 +13,7 @@ A [policy](../kyverno-policies) contains one or more rules, and the following co
 
 * **background**: controls scanning of existing resources to find potential violations and generating Policy Reports. See the documentation [here](../policy-reports/background.md). Defaults to "true".
 
-* **failurePolicy**: defines the API server behavior if the webhook fails to respond. Allowed values are "Ignore" or "Fail". Defaults to "Fail". Additionally, if set to "Ignore" will allow failing calls to image registries to be ignored. This allows for rule types like verifyImages or others which use image data to not block if the registry is temporarily down, useful in situations where images already exist on the nodes.
+* **failurePolicy**: defines the API server behavior if the webhook fails to respond. Allowed values are "Ignore" or "Fail". Defaults to "Fail". Additionally, if set to "Ignore" will allow failing calls to image registries to be ignored. This allows for rule types like verifyImages or others which use image data to not block if the registry is temporarily down, useful in situations where images already exist on the nodes. This field is deprecated as of 1.13. Scheduled to be removed in a future version.
```

Comment:
Can we clarify the new field which replaces this existing failurePolicy? Same for the following 3 deprecated fields.